.github/workflows/pull-request-ci.yaml

```diff
@@ -199,14 +199,21 @@ jobs:
           echo "::set-env name=SSH_AUTH_SOCK::${SSH_AUTH_SOCK}"
           echo "::set-env name=SSH_AGENT_PID::${SSH_AGENT_PID}"
 
-      # set GKE secret. TBD: use vault instead
+      - name: Query vault for GKE secrets
+        id: vault
+        uses: hashicorp/vault-action@v2.0.1
+        with:
+          url: https://volt.cap.explore.suse.dev/
+          token: ${{ secrets.VAULT_TOKEN }}
+          secrets: |
+            secret/data/cloud-providers "gke.GKE_CRED_JSON" | GKE_CRED_JSON
```

Comment:
We could possibly set `exportEnv` to `false`, and then use the output explicitly in the next step, to ensure that we don't accidentally leak the contents later.  Not that it matters that much, since people can read the file… but it helps to avoid accidentally printing it out when dumping the environment.

Reply:
See commit f3c7c8d8411a3c291ca677ec5be23283aa41c778